packages/cli/src/applyPreset.ts

```diff
@@ -0,0 +1,54 @@
+// MUST import this file first before anything and not import any Lodestar code
```

Comment:
Can you add more comment about motivation here? Or link to this PR/issue if we want to keep motivation text there?

Reply:
Done